tools/shared.py

```diff
@@ -2707,8 +2707,9 @@ def path_to_system_js_libraries(library_name):
       'c': '',
       'dl': '',
       'EGL': 'library_egl.js',
-      'GL': 'library_gl.js',
-      'GLESv2': 'library_gl.js',
+      'GL': 'library_webgl.js',
+      'GLESv2': 'library_webgl.js',
+      'GLESv3': ['library_webgl.js', 'library_webgl2.js'],
```

Comment:
Maybe I've missed something, but I think OpenGL ES 3.0+ are traditionally provided by libGLESv2.so instead of libGLESv3.so?

Reply:
Oh yeah, that's right. Updated.